lib/v2/coronavirus/tg-nCoV2019.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+//const dateParser = require('@/utils/dateParser');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+	const district = ctx.params.district;
+	const queryUrl = district ? `https://t.me/s/nCoV2019?q=%23${encodeURIComponent(district)}` : `https://t.me/s/nCoV2019`;
+	const response = await got.get(queryUrl);
+	const $ = cheerio.load(response.data);
+	const list = $('.tgme_widget_message_bubble');
+	
+	if (list.length === 0 && $('.tgme_channel_history').length === 0) {
+        throw `Unable to fetch message feed from this channel. Please check this URL to see if you can view the message preview: https://t.me/s/${username}`;

```

Comment:
```suggestion
        throw 'Unable to fetch message feed from this channel. Please check this URL to see if you can view the message preview: https://t.me/s/nCoV2019';
```

Reply:
Thanks, will be more attentive next time 😉